module-services/service-desktop/parser/ParserFSM.cpp

```diff
@@ -7,20 +7,32 @@
 #include <memory>
 #include <string>
 
+#include <Timers/TimerFactory.hpp>
 #include "MessageHandler.hpp"
 #include <endpoints/EndpointFactory.hpp>
 #include <endpoints/message/Common.hpp>
 
+namespace
+{
+    constexpr auto parserTimerDelayMs = std::chrono::milliseconds{1000 * 5};
```

Comment:
Shouldn't it be like `connectionTimeout` or ` responseTimeout` ?

Reply:
Fixed